mono/mini/interp/interp.c

```diff
@@ -3361,10 +3360,11 @@ method_entry (ThreadContext *context, InterpFrame *frame,
  * FRAME is only valid until the next call to alloc_frame ().
  */
 static MONO_NEVER_INLINE void
-interp_exec_method (InterpFrame *frame, ThreadContext *context, FrameClauseArgs *clause_args, MonoError *error)
+interp_exec_method (InterpFrame *frame, ThreadContext *context, FrameClauseArgs *clause_args)
 {
 	InterpMethod *cmethod;
 	MonoException *ex;
+	ERROR_DECL(error);
```

Comment:
This argument was added to reduce native frame size, MonoError is huge.

Reply:
I did shrink MonoError some, and this does not consume wasm stack because it is address-taken, and the non-recursion makes it matter less. But yes, could still move to caller.